app/models/form/lettings/pages/housing_provider.rb

```diff
@@ -0,0 +1,30 @@
+class Form::Lettings::Pages::HousingProvider < ::Form::Page
+  def initialize(id, hsh, subsection)
+    super
+    @id = "housing_provider"
+    @header = ""
+    @description = ""
+    @subsection = subsection
+  end
+
+  def questions
+    @questions ||= [
+      Form::Lettings::Questions::HousingProvider.new(nil, nil, self),
+    ]
+  end
+
+  def routed_to?(log, current_user)
+    return false unless current_user
+    return true if current_user.support?
+    return true unless current_user.organisation.holds_own_stock?
+
+    housing_providers = current_user.organisation.housing_providers
+
+    return false if housing_providers.count.zero?
+    return true if housing_providers.count > 1
+
+    log.update!(owning_organisation: housing_providers.first)
```

Comment:
What if the org holds it's own stock and has 1 housing provider? Could it not be an owning organisation of a log?

Reply:
Couldn't the org be its own `owning_organisation `for this log in that case?